CLAUDE.md

```diff
@@ -0,0 +1,23 @@
+# PocketSizeFund Assistant Memory
+
+## GitHub Issue Creation
+
+When asked to create a new GitHub issue:
+
+1. Follow the structure in `.github/ISSUE_TEMPLATE.md`
```

Comment:
This isn't the right filepath - there's a directory called `ISSUE_TEMPLATE` with the `BUG.md` and `FEATURE.md` files.